pkg/main.go

```diff
@@ -61,13 +66,22 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	opts := zap.Options{
-		Development: true,
-	}
-	opts.BindFlags(flag.CommandLine)
+
 	flag.Parse()
 
-	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
+	var logger logr.Logger
```

Comment:
doing so, would generate logs that have the actual line number of source code,  eg:
```
2022-02-14T16:57:00.567-0500	INFO	controller.hypershiftdeployment	controllers/hypershiftdeployment_controller.go:91	Reconcile: clusters/hyper-pr	{"reconciler group": "cluster.open-cluster-management.io", "reconciler kind": "HypershiftDeployment", "name": "hyper-pr", "namespace": "clusters"}
```

I feel this is much useful when debugging